cabal-testsuite/cabal-testsuite.cabal

```diff
@@ -26,10 +26,10 @@ common shared
   default-language: Haskell2010
 
   build-depends:
-    , base >= 4.6 && <4.17
+    , base >= 4.6 && <4.18
     -- this needs to match the in-tree lib:Cabal version
-    , Cabal ^>=3.8.0.0
-    , Cabal-syntax ^>=3.8.0.0
+    , Cabal ^>=3.8.1.0
+    , Cabal-syntax ^>=3.8.1.0
```

Comment:
We actually keep these, and everywhere else, at X.X.0.0 to avoid having to update at each minor release.

Reply:
Okay!